code/datums/uplink_item.dm

```diff
@@ -593,14 +593,14 @@ var/list/uplink_items = list()
 	name = "Pizza Bomb"
 	desc = "A pizza box with a bomb taped inside it. The timer needs to be set by opening the box; afterwards, opening the box again will trigger the detonation."
 	item = /obj/item/device/pizza_bomb
-	cost = 6
+	cost = 5
```

Comment:
It ain't gonna gib i think, only kill/crit, but putting the cost to 5 means that 4 of them are buyable so ehh, 6 seems better